Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.py

```diff
@@ -411,7 +414,7 @@ def main():
     tenant_id: str = params.get('credentials_tenant_id', {}).get('password') or params.get('tenant_id', '')
     auth_and_token_url: str = params.get('credentials_auth_id', {}).get('password') or params.get('auth_id', '')
     enc_key: str = params.get('credentials_enc_key', {}).get('password') or params.get('enc_key', '')
-    base_url: str = urljoin(params.get('url', ''), '/v1.0')
```

Comment:
I did not delete anything in the code, I just changed the location.
Now it is in the client, because I get the final url only there.
`self.base_url = urljoin(self.azure_cloud.endpoints.microsoft_graph_resource_id, '/v1.0')`